sh/fish/tabry_fish.fish

```diff
@@ -10,15 +10,21 @@ function tabry_completion_init
   complete -c "$cmd" -n "__fish_tabry_check_args_and_file" -a "(__fish_tabry_completions)"
 end
 
+# _tabry_#{uniq_fn_id}_completions() {
+#   TABRY_IMPORTS_PATH=#{esc import_path} _tabry_#{uniq_fn_id}_completions_internal #{esc tabry_bash_executable} #{tabry_bash_arg && esc(tabry_bash_arg)}
+# }
+
```

Comment:
Can/should this be removed or did you intend it for documentation purposes?

Reply:
Ah, I forgot I had that in there to copy/compare, just removed it